akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala

```diff
@@ -21,7 +21,7 @@ import akka.util.Helpers.ConfigOps
 
 /**
  * Implementation of 'The Phi Accrual Failure Detector' by Hayashibara et al. as defined in their paper:
- * [http://www.jaist.ac.jp/~defago/files/pdf/IS_RR_2004_010.pdf]
+ * [https://www.jaist.ac.jp/~defago/files/pdf/IS_RR_2004_010.pdf]
```

Comment:
Same for this one

Reply:
I was able to find the up-to-date link for this and updated the same.